.github/workflows/kgw-test-reuse.yaml

```diff
@@ -0,0 +1,179 @@
+name: kgw-test-reuse
+
+on:
+  workflow_call:
+    inputs:
+      # for logging purpose
+      kdb-repo:
+        type: string
+      # for workflow_run trigger, we need to check out to the sha, bcz most of
+      # the pr come from forked repo, and we can't access the branch, but we can
+      # access the sha(i.e, pull/xxx/head)
+      kdb-sha:
+        type: string
+      kdb-branch:
+        required: true
+        type: string
```

Comment:
Is this current with main's?  `required` is true for branch one but can't it work with sha?  Otherwise PR looks good to me.

Reply:
Oh, it is current.  I'm thinking of manual kgw test....